package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "plywood",
-  "version": "0.26.9",
+  "version": "0.26.10",
```

Comment:
I think we use `np` to publish these packages, which would take care of the version bump. The general process would be:

1. Make your changes, open the PR
2. Merge the PR
3. Run `np` locally to bump the version and publish the package